elements/designer-element-palette/designer-element-palette.html

```diff
@@ -64,10 +59,16 @@
     <h1>Elements</h1>
     <div id="elements">
       <template is="dom-repeat" items="{{builtIns}}">
-        <div class="element" on-mousedown="_onMouseDown">&lt;<span>{{item}}</span>&gt;</div>
+        <div class="element" on-mousedown="_onMouseDown">
+          &lt;<span>{{item}}</span>&gt;
+        </div>
       </template>
     </div>
-    <div id="proxy"></div>
+    <div id="proxy"
+        on-mouseover="_onProxyMouseOver"
+        on-mouseout="_onProxyMouseOut"
+        on-mouseup="_onProxyMouseUp">
```

Comment:
That might be very handy, if it works for all my needs. it would mean a rewrite, and possible serious pruning of my dragging utility code, so let's defer to later PR.